lib/vdsm/gluster/cli.py

```diff
@@ -1142,24 +1142,42 @@ def _parseGeoRepStatus(tree):
                ]....
     }
     """
+    if tree.find('geoRep/volume/sessions/session/session_secondary') is None:
+        # Gluster version < 10
+        gluster_format = 9
+    else:
+        # Gluster version >= 10
+        gluster_format = 10
```

Comment:
Take this comment as a suggestion.

This works, but it is a boolean in the end, so it could be:
```python
gluster_old_format = tree.find('geoRep/volume/sessions/session/session_secondary') is None
```
If we need to support more formats in the future, we could use global variables for them. But boolean suffices for now. And then below we just need:
```python
if gluster_old_format:
     ...
else:
     ...
```
`elifs` check the condition again, which is not necessary for now.

Reply:
Done.